app/containers/InterventionDetailsPage/containers/ParticipantInviter/CopyLinkForm.tsx

```diff
@@ -23,7 +20,7 @@ export type Props = {
   isReportingIntervention: boolean;
   interventionId: string;
   interventionLanguageCode: string;
-  sessionOptions: SelectOptionTypeExtended<string>[];
+  sessionOptions: (SelectOption<string> & { type: string })[];
```

Comment:
You don't need that here

```suggestion
  sessionOptions: SelectOption<string>[];
```